consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/RoundState.java

```diff
@@ -94,10 +99,15 @@ public void addCommitMessage(final SignedData<CommitPayload> msg) {
   private void updateState() {
     // NOTE: The quorum for Prepare messages is 1 less than the quorum size as the proposer
     // does not supply a prepare message
-    prepared =
-        (preparePayloads.size() >= prepareMessageCountForQuorum(quorum))
-            && proposalMessage.isPresent();
+    final long prepareQuorum = prepareMessageCountForQuorum(quorum);
```

Comment:
this should get moved into the class constructor (i.e. can be calculated at startup).